### DISCUSSION TITLE
Why does the create-account endpoint return status 200 Ok instead of 201 Created?

### DISCUSSION THREAD

**Discussion by pboling** [jeremyevans/rodauth#385] [General]
The title.

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

Not suggesting it is wrong, rather I'm assuming there is a reason, and I'm trying to learn from smart people. :)

**Comment by jeremyevans** [comment]
Rodauth doesn't return the newly created account in the body of the message, the created account is not located at the URL of the request, and the Location header probably does not refer to the URL of the created account (Rodauth doesn't offer a URL for accounts themselves).

**Reply by pboling** [reply]
Perfect.  Thank you!
